app/assets/javascripts/services/miq_service.js

```diff
@@ -36,6 +36,10 @@ ManageIQ.angular.app.service('miqService', ['$timeout', '$document', '$q', 'API'
     return miqJqueryRequest(url, options);
   };
 
+  this.refreshSelectpicker = function() {
+    $('select').selectpicker('refresh');
```

Comment:
This can break on any screen using selects which are not bootstrap-selectpicker - the `select` is a bit too liberal as a selector.

(But *probably* ok since dialog-user screens don't have any other content in them, right?)

Reply:
Yes, any select for a dialog-user should be a patternfly selectpicker style. You're right it's a bit liberal, I was mainly focusing on getting this out as a resolution to a high priority BZ, but we can probably change the selector to be more conservative.